docs/configuration.md

```diff
@@ -2397,6 +2397,14 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>1.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.network.timeoutInterval</code></td>
+  <td>60s</td>
```

Comment:
Check `spark.rpc.io.connectionCreationTimeout`, `spark.rpc.io.connectionTimeout`... and `park.rpc.askTimeout`. All of them referencing to another config where the default comes from.

For  `spark.network.timeoutInterval` the default is:
https://github.com/apache/spark/blob/418bba5ad6053449a141f3c9c31ed3ad998995b8/core/src/main/scala/org/apache/spark/internal/config/Network.scala#L48-L52

So IMHO we should mention here `spark.storage.blockManagerTimeoutIntervalMs` as the default.

Reply:
Oh I see those are not hardcoded values. In this case it is fine!